app/src/main/java/com/example/hangouts/homeScreen/HomeFragment.java

```diff
@@ -0,0 +1,53 @@
+package com.example.hangouts.homeScreen;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.example.hangouts.R;
+import com.example.hangouts.databinding.FragmentHomeBinding;
+import com.example.hangouts.models.User;
+import com.parse.ParseUser;
+
+public class HomeFragment extends Fragment {
+
+    private FragmentHomeBinding binding;
+
+    private TextView tvUserName;
+    private ParseUser currentUser;
+
+    public HomeFragment() {}
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        binding = FragmentHomeBinding.inflate(getLayoutInflater());
+        return binding.getRoot();
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        currentUser = ParseUser.getCurrentUser();
+
+        tvUserName = binding.tvUserName;
+        tvUserName.setText(String.format("%s %s.",
+                currentUser.get(User.KEY_NAME).toString(),
+                currentUser.get(User.KEY_LASTNAME).toString().substring(0, 1)));
```

Comment:
What happens if the Last name is an empty string?

Reply:
Suggestion - you may have more code like this that is related to the user model.  It's a good idea to consolidate this code in some way to prevent duplication, separate concerns, and increase modularization.  Seems trivial and small now, but can snowball and bugs can creep in.  There are multiple ways to do this.
 1 - A separate user model class for your UI which is created from the parse user
 2 - add methods to the Parse User model - i.e. `getLastInitial()`
 3 - a Utils static class where you pass in the user model - i.e. `UserUtils.getLastInitial(currentUser)
 
 I favor #1 generally because it allows separating UI domain from the backend domain.  However for this project I would recommend 2 or 3 as it is more straight forward and less code.